builder/builder-next/builder.go

```diff
@@ -313,10 +313,24 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 	}
 	frontendAttrs["add-hosts"] = extraHosts
 
+	exporterName := ""
 	exporterAttrs := map[string]string{}
 
-	if len(opt.Options.Tags) > 0 {
-		exporterAttrs["name"] = strings.Join(opt.Options.Tags, ",")
+	if len(opt.Options.Outputs) > 1 {
+		return nil, errors.Errorf("multiple outputs not supported")
+	} else if len(opt.Options.Outputs) == 0 {
+		exporterName = "moby"
+	} else {
+		if opt.Options.Outputs[0].Type != "cacheonly" {
+			exporterName = opt.Options.Outputs[0].Type
+			exporterAttrs = opt.Options.Outputs[0].Attrs
+		}
+	}
+
+	if exporterName == "moby" {
```

Comment:
Wondering we should call it `docker` for BuildKit API compatibility

Or `internal`

Reply:
I don't want to call it `docker` cause that already means a different exporter. This value is completely invisible to user.

Reply:
It is also temporary. When containerd support lands, this should all go away.

Reply:
So, on client side, it is prohibited to set the output type to `moby` explicitly?

Reply:
Currently, you could use it to override options but I would never document it as a guaranteed behavior.